### ISSUE TITLE
v2.1.3 Failed to mount component: template or render function not defined

### ISSUE THREAD

**Issue by adavie1** [MatteoGabriele/vue-progressive-image#18]
This may be me being stupid (quite likely :-D), but I'm getting the above error.

### Expected behavior
```
<template>
<div>
  ...

  <progressive-background src="...">
    <!-- content to display -->
</progressive-background>

 ...
  </div>
</template>

<script>
import ProgressiveBackground from 'vue-progressive-image'

export default {
  components: {
    ProgressiveBackground
  },
...

</script>
```
Should display the progressive-background image with the content on top.

As a side note, I prefer not to use `Vue.use(...)` as a general rule. Only used for UI frameworks that are used throughout the app.

I've tried with `Vue.use(ProgressiveBackground)` still get the same issue.

### Actual behavior

Getting the error above

### Environment

Run this command in the project folder and fill in the result:

`npm ls vue-progressive-image`: 2.1.3

Then, specify:

1. Operating system: Linux
2. Browser and version: Chrome v60

### Reproducible Demo

Error prevents that.

**Comment by MatteoGabriele** [comment]
Hi @adavie1 
well you should use the `use` method because that one installs the components: I don't create 2 single components file but just one file that holds them both. It's mandatory.

For the other problem of the content not visible, this is fixed with the 2.1.3 ( latest release )
If after a normal installation still doesn't work, please create a reproduction of this issue and re-open this ticket.

Thanks for your feedback :)

**Comment by adavie1** [comment]
Noted. All working OK now. Thanks
